aws_lambda_powertools/utilities/data_classes/dynamo_db_stream_event.py

```diff
@@ -1,15 +1,39 @@
 from enum import Enum
-from typing import Dict, Iterator, List, Optional
+from typing import Any, Dict, Iterator, List, Optional, Union
 
 from aws_lambda_powertools.utilities.data_classes.common import DictWrapper
 
 
+class AttributeValueType(Enum):
```

Comment:
Could you update the DynamoDB Stream data class doc to include an example usage?

Reply:
Ok sure.

Reply:
@heitorlessa CI process seems to be broken